verify/pylint_bin.py

```diff
@@ -32,8 +31,10 @@
             if real != full:
                 EXTRAS.add(os.path.dirname(real))
                 break
-    for extra in EXTRAS:
-        sys.path.append(extra)
+    # also do one level up so foo.bar imports work :shrug:
+    EXTRAS = set(os.path.dirname(e) for e in EXTRAS).union(EXTRAS)
+    # append these to the path
+    sys.path.extend(EXTRAS)
```

Comment:
isn't it you want to put them before the existing `sys.path`?

Reply:
You'd think so but somehow that causes pylint to give impossible errors on some `# pylint: disable=foo` lines

Reply:
It gives bad-config-value if you attempt to disable or enable import-error or wrong-import-order.. I have no idea why